File: tests/fixtures/language_pack/src/ruby/widget.rb
require "json"

RUBY_LIMIT = 4

module RubyPack
  class RubyWidget
    def render
      ignored = "class RubyStringFake; end"
      ignored
    end

    def self.build_ruby_widget(
      name
    )
      new
    end
  end
end

File: tests/fixtures/language_pack/src/swift/Widget.swift
import Foundation

typealias SwiftName = String

protocol SwiftRenderable {
    func render(
        prefix: String
    ) -> String
}

enum SwiftMode {
    case compact
}

struct SwiftPayload {
    let value: String
}

class SwiftWidget: SwiftRenderable {
    let SwiftLimit = 4
    let name: SwiftName

    init(
        name: SwiftName
    ) {
        self.name = name
    }

    func render(
        prefix: String
    ) -> String {
        let ignored = "class SwiftStringFake"
        return prefix + name
    }
}

func buildSwiftWidget(
    name: SwiftName
) -> SwiftWidget {
    return SwiftWidget(name: name)
}
